lib/cocoapods/installer/xcode/pods_project_generator/target_installer.rb

```diff
@@ -106,19 +108,27 @@ def custom_build_settings
           #
           # Saves the content the provided path unless the path exists and the contents are exactly the same.
           #
-          # @return [Object] the result of the generator.
```

Comment:
The return value from here was never actually used anywhere

Reply:
it was being used but I think it no longer did due to the build settings re-write.